CONTRIBUTORS_OLD.md

```diff
@@ -15299,6 +15299,7 @@
 
 - [@mikeescro](https://github.com/mikeescro)
 
-=======
 - [@Eamos2] (https://github.com/Eamos2)
 >>>>>>> c90cc8dfb (added name to contribtor list)
+- [@emerald211](https://github.com/emerald211)
+=======
```

Comment:
You've applied your line addition on the wrong folder, please read again the README.md file to know where to add your contribution.
Once done, please remove any changes on this current file.
Until adjustments, this cannot be merged.